paas/client.go

```diff
@@ -474,7 +474,20 @@ func (c *CarrierClient) createRepo(name string) error {
 }
 
 func (c *CarrierClient) createRepoWebhook(name string) error {
-	c.ui.Normal().Msg("Creating webhook in the repo ...")
+	hooks, resp, err := c.giteaClient.ListRepoHooks(c.config.Org, name, gitea.ListHooksOptions{})
+	if resp == nil && err != nil {
+		return errors.Wrap(err, "failed to list webhooks")
```

Comment:
Not sure if that can even happen, but if resp is not nil and we have an error, then this code ignores the error. Maybe just:

```
if err != nil {
  return errors.Wrap(err, "failed to list webhooks")
}
```

Reply:
thanks, TIL :smile: